stable/rabbitmq/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 6.9.0
+version: 6.9.1
```

Comment:
As this is a new feature, the version should be incremented in a minor (`6.10.0`).

Reply:
Makes sense - updated to `6.10.0` in dfcf104200844279ec03f7c4ee199419cb102297